person/models.py

```diff
@@ -1,14 +1,33 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 from django.utils import timezone
+from django.core.validators import MinValueValidator
+from django.core.validators import RegexValidator
 
 SEX_TYPES = (
     ('F','Femenino'),
     ('M', 'Masculino'),
     ('O', 'Otro'),
 )
 
-class Person(AbstractUser):
+PAYMENT_METHOD = (
+    ('T', 'Transferencia'),
+    ('TB','Tarjeta Bancaria')
+)
+
+STATUS = (
+    ('C', 'Casado/a'),
+    ('F', 'Fallecido/a'),
+    ('V', 'Viudo/a'),
+    ('S', 'Soltero/a')  
+)
+
+FREQUENCY = (
```

Comment:
I would also include "Trimestral" and "Semestral", since there are people who decide to stop for one of these periods of time.

Reply:
okey